### DISCUSSION TITLE
✅ Run Command on ssh connection

### DISCUSSION THREAD

**Discussion by mbierman** [isontheline/pro.webssh.net#349] [Ideas]
Maybe I’m imagining things, but didn’t it used to be possible to run a command on ssh connection? I don’t see it anymore. This would be really nice to have.

**Comment by isontheline** [comment]
Your connection > (i) > Terminal Settings > Execute command

**Reply by mbierman** [reply]
Ah! I missed it. Thank you.

**Reply by isontheline** [reply]
You're Welcome 😁
